integration/query_test.go

```diff
@@ -950,7 +950,11 @@ func TestQueryBatchSize(t *testing.T) {
 
 func TestQueryCommandGetMore(t *testing.T) {
 	t.Parallel()
-	ctx, collection := setup.Setup(t)
+	s := setup.SetupWithOpts(t, &setup.SetupOpts{
+		ClientOptions: options.Client().SetMaxPoolSize(1),
```

Comment:
At the very least, we should also set `SetMinPoolSize` to prevent the number of active connections from dropping to zero. Please also check other options.

Reply:
Good idea, applied in the tests.